data/coalition missions.txt

```diff
@@ -18,6 +18,7 @@ mission "Coalition: First Contact"
 	on offer
 		"reputation: Coalition" += 10
 		"reputation: Heliarch" += 10
+		log "Made contact with the Coalition, an alliance of giant beetles, giant spiders and centaurs in the galactic south. They're ruled by 'Heliarchs', and they overthrew the Quarg. Seems they see them as oppressors."
```

Comment:
We're an Oxford comma lot, eh? I seem to vary my use of it depending on how I hear the sentence pronounced. Totally fine with this though, and I've merged in your changes. @jafdy, great catch.